socorro/external/rabbitmq/crashstorage.py

```diff
@@ -299,3 +299,29 @@ class ReprocessingRabbitMQCrashStore(RabbitMQCrashStorage):
         False
     )
 
+
+#==============================================================================
+class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
+    required_config = Namespace()
+    required_config.rabbitmq_class = change_default(
+        RabbitMQCrashStorage,
+        'rabbitmq_class',
+        ConnectionContext,
+    )
+    required_config.routing_key = change_default(
+        RabbitMQCrashStorage,
+        'routing_key',
+        'socorro.reprocessing'
+    )
+
+    def __init__(self, config, quit_check_callback=None):
+        super(ReprocessingRabbitMQCrashStore, self).__init__(
+            config, quit_check_callback=quit_check_callback
+        )
```

Comment:
It actually calls the parent's parent, since the class is named ``ReprocessingOneRabbitMQCrashStore`` and its parent is named ``ReprocessingRabbitMQCrashStore``. Note that the first argument to ``super`` is the latter, so what you're calling is the ``__init__`` of the parent of ``ReprocessingRabbitMQCrashStore`` (without 'One'). 

Or I completely misunderstood how ``super`` works, in which case you have to tell me because that would be quite a flow in my education. :)